recipes/augustus/3.2.2/meta.yaml

```diff
@@ -0,0 +1,119 @@
+about:
+  home: http://bioinf.uni-greifswald.de/augustus/
+  license: ''
+  summary: 'AUGUSTUS is a gene prediction program for eukaryotes written by Mario
+    Stanke and Oliver Keller. It can be used as an ab initio program, which means
+    it bases its prediction purely on the sequence. AUGUSTUS may also incorporate
+    hints on the gene structure coming from extrinsic sources such as EST, MS/MS,
+    protein alignments and synthenic genomic alignments.'
+build:
+  number: 0
+package:
+  name: augustus
+  version: '3.2.2'
+  skip: True # [osx]
+requirements:
+  build:
+      - libgcc
+      - gcc
+      - bamtools
+      - perl-app-cpanminus
+      - perl-yaml
+      - perl-dbi
+      - boost
```

Comment:
@bgruening 

When I try to pin the boost dep I get the following error on building

`
Extracting download
Applying patch: '/bioconda-recipes/recipes/augustus/3.2.2/patches/bam2hints.Makefile.patch'
patching file auxprogs/bam2hints/Makefile
Applying patch: '/bioconda-recipes/recipes/augustus/3.2.2/patches/filterBam.src.Makefile.patch'
patching file auxprogs/filterBam/src/Makefile
Applying patch: '/bioconda-recipes/recipes/augustus/3.2.2/patches/filterBam.Makefile.patch'
patching file auxprogs/filterBam/Makefile
Error: Failed to render jinja template in /bioconda-recipes/recipes/augustus/3.2.2/meta.yaml:
'CONDA_BOOST' is undefined
`

I just did a docker build, so I don't think that's the issue.